src/Carbon/Carbon.php

```diff
@@ -112,6 +112,11 @@ class Carbon extends DateTime
     const SECONDS_PER_MINUTE = 60;
 
     /**
+     * RFC7231 DateTime format.
+     */
+    const RFC7231 = 'D, d M Y H:i:s \G\M\T';
```

Comment:
This is the case for all built-in `Datetime` formats.
The fomats which are related to `Carbon` only are suffixed with `_FORMAT`: see `DEFAULT_TO_STRING_FORMAT`